packages/office-ui-fabric-react/src/components/GroupedList/GroupedListSection.tsx

```diff
@@ -222,9 +223,12 @@ export class GroupedListSection extends React.Component<IGroupedListSectionProps
     const groupShowAllProps: IGroupShowAllProps = { ...showAllProps, ...dividerProps };
     const groupFooterProps: IGroupFooterProps = { ...footerProps, ...dividerProps };
 
+    const isDraggable: boolean = !!this.props.dragDropHelper && this._getGroupDragDropOptions().canDrag!(group);
```

Comment:
Could you describe why this new isDraggable const is being introduced? More detailed description on this whole change box in the PR would help.